Web/starter_project_barbies/pages/blog/BlogDetail.tsx

```diff
@@ -0,0 +1,137 @@
+import BlogCard from '@/components/blog/BlogCard'
+import fonts from '@/styles/blog/blog-detail.module.css'
+import { Blog } from '@/types/blog'
+import Image from 'next/image'
+
+interface BlogDetailProps {
+  blog: Blog
+}
+
+interface BlogAuthorDetailProps {
```

Comment:
[nit] this is fine but i would prefer if each prop is above it's own component for a bit more consistency and clarity